task_12/src/main.scss

```diff
@@ -0,0 +1,216 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,700');
+@import "reset";
+@import "variables";
+
+$fa-font-path: "~font-awesome/fonts";
+@import "~font-awesome/scss/font-awesome.scss";
+
+.button {
+	border: none;
+	border-radius: 5px;
+	padding: 1rem 1.5rem;
+	min-height: 40px;
+	text-transform: uppercase;
+	color: #fff;
+	font-size: 1.6rem;
+	white-space: nowrap;
+	transition: translform 0.1s,
+							box-shadow $transition;
+	cursor: pointer;
+	&:hover {
+		box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2),
+								0 6px 20px 0 rgba(0,0,0,0.19);
+	}
+	&:active {
+		transform: translateY(1px);
+	}
+	&:focus {
+		outline-color: $color-red;
+	}
+}
+
+a:focus {
+	outline-color: $color-red;
+}
+
+html {
+	box-sizing: border-box;
+	min-width: 320px;
+	font-size: 10px;
+	height: 100%;
+	background: $color-orange-red;
+}
+*,
+*::before,
+*::after {
+	box-sizing: inherit;
+}
+
+body {
+	font-family: #{$font}, sans-serif;
```

Comment:
It's better to use shorthand in html instead of splitting font configuration into 2 selectors.